src/diffusers/loaders.py

```diff
@@ -1021,6 +1082,8 @@ def load_lora_into_text_encoder(cls, state_dict, network_alpha, text_encoder, lo
             lora_scale (`float`):
                 How much to scale the output of the lora linear layer before it is added with the output of the regular
                 lora layer.
+            state_dict_aux (`dict`, *optional*):
+                A dictionary containing the auxilary state dict (additional lora state) for the text encoder.
```

Comment:
```suggestion
            state_dict_aux (`dict`, *optional*):
                A dictionary containing the auxilary state dict (additional lora state) for the text encoder.
```

Why do we need an additional dict here - can't we have this directly in the passed `state_dict`?